src/pages/pattern/pattern-details/pattern-details.js

```diff
@@ -0,0 +1,288 @@
+            {checkboxInputs}
+            <Grid item xs={12}>
+              <Paper className={classes.patternItemUpdate}>
+                <TextField
+                  id='large'
+                  className={classes.textField}
+                  variant='outlined'
+                  label='Фото великого розміру'
+                  multiline
+                  value={large}
+                  onChange={(e) => setLarge(e.target.value)}
+                  required
+                />
+                <TextField
+                  id='medium'
+                  className={classes.textField}
+                  variant='outlined'
+                  label='Фото середнього розміру'
+                  multiline
+                  value={medium}
+                  onChange={(e) => setMedium(e.target.value)}
+                  required
+                />
+                <TextField
+                  id='small'
+                  className={classes.textField}
+                  variant='outlined'
+                  label='Фото малого розміру'
+                  multiline
+                  value={small}
+                  onChange={(e) => setSmall(e.target.value)}
+                  required
+                />
+                <TextField
+                  id='thumbnail'
+                  className={classes.textField}
+                  variant='outlined'
+                  label='Фото найменшого розміру'
+                  multiline
+                  value={thumbnail}
+                  onChange={(e) => setThumbnail(e.target.value)}
+                  required
+                />
+                <TextField
+                  id='material'
+                  className={classes.textField}
+                  variant='outlined'
+                  label='Матеріал'
+                  multiline
+                  value={material}
+                  onChange={(e) => setMaterial(e.target.value)}
+                  required
+                />
+              </Paper>
+            </Grid>
+            <Grid item xs={6}>
+              <Paper className={classes.patternItemUpdate}>
+                <TextField
+                  id='ukName'
+                  className={classes.textField}
+                  variant='outlined'
+                  label='назва (укр.)'
+                  multiline
+                  value={ukName}
+                  onChange={(e) => setUkName(e.target.value)}
+                  required
+                />
+                <TextField
+                  id='ukDescription'
+                  className={classes.textField}
+                  variant='outlined'
+                  label='опис (укр.)'
+                  multiline
+                  value={ukDescription}
+                  onChange={(e) => setUkDescription(e.target.value)}
+                  required
+                />
+              </Paper>
+            </Grid>
+            <Grid item xs={6}>
+              <Paper className={classes.patternItemUpdate}>
```

Comment:
Please, move this component `Paper` and its inner components to the separate component to reduce size of current file and improve readability

Reply:
refactored

Reply:
fixed